src/Microsoft.ML.CodeGenerator/Templates/Console/PredictProject.tt

```diff
@@ -54,4 +62,5 @@ public bool IncludeResNet18Package {get; set;}
 public bool IncludeRecommenderPackage {get;set;}
 public string StablePackageVersion {get;set;}
 public string UnstablePackageVersion {get;set;}
+internal CSharp.GenerateTarget Target {get;set;}
```

Comment:
NIT: I would think of CLI vs ModelBuilder as the Source instead of Target. It's the thing causing the source to get generated.

Reply:
Will do